casepro/msgs/views.py

```diff
@@ -26,6 +29,22 @@
 RESPONSE_DELAY_WARN_SECONDS = 24 * 60 * 60  # show response delays > 1 day as warning
 
 
+# Override the ImportTask start method so we can use our self-defined task
+def override_start(self, org):
+    from .tasks import faq_csv_import
+    self.log("Queued import at %s" % now())
+    self.save(update_fields=['import_log'])
+
+    # trigger task
+    result = faq_csv_import.delay(org, self.pk)
+    # TODO: celery setup required? currently always_eager set to true
```

Comment:
I'm actually not sure - it didn't look to me like it was necessary to update any settings, but your knowledge of celery is much better than mine - could you have a look if I overlooked something?